src/ray/gcs/gcs_client/accessor.cc

```diff
@@ -19,13 +19,6 @@
 #include "ray/common/asio/instrumented_io_context.h"
 #include "ray/gcs/gcs_client/gcs_client.h"
 
-namespace {
-inline int64_t GetGcsTimeoutMs() {
```

Comment:
This function is still used inside this class. Are we importing it from gcs_client instead?